README.md

```diff
@@ -1,30 +1,148 @@
-# Edu Client
```

Comment:
I am really liking your readme, and all the emojis and everything, but it is too verbose.

You can create a new PR once PR https://github.com/codeforcauseorg/edu-client/pull/23 is merged to put your changes on top of it. 

For now save it somewhere else, and remove this change.

Reply:
@Siddhant-K-code Told me the structure according to that I created the readme, What should I do?

Reply:
No it's all fine, the readme is really looking beautiful, I am telling that please remove the changes of readme from this PR since https://github.com/codeforcauseorg/edu-client/pull/23 is already about to merge, so once that will be merged you can put your further changes after that.

For now remove all the readme changes from this PR.

Reply:
Because according to him content in #23 pr is not prioritized corretly.

Reply:
ok

Reply:
Then I should only keep contributing.md for now?

Reply:
Yes

Reply:
@Abhishek-kumar09 created PR with a lot of details, also he also informed first, #23 was with no assignment and also #23 has less info.

Reply:
Yes as said @Abhishek-kumar09  I have created a pull request with only contributing.md. Please let me know what to do.

Reply:
@Siddhant-K-code , @Abhishek-kumar09  Also I have documented the changes made in package.json, package-lock.json, and yarn.lock so that the user will not encounter issues further.